### PR TITLE
Add current web accessibility trends question to accessibility-quiz.ts

### PR DESCRIPTION
**Pull request by stmapman7**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=ivdmUgZ)
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=9-iMBz5)
Fixed a few grammar errors and added a web accessibility current trends question for 2023.

## Summary of changes

I made a few spelling and grammar changes to some of the quiz questions and also added one concerning current web accessibility trends for 2023.

## Checklist

- [X] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [X] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [X] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [X] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [X] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -550,6 +550,16 @@ const accessibilityQuiz = [
...
   },
+  {
+    Question: "Which is not a current web accessibility trend?",
+    Answer: "GDPR-compliant websites",
+    Distractor1: "Virtual and augmented reality technologies",
+    Distractor2: "Utilizing Machine Learning",
+    Distractor3: "Search Engine Optimization (SEO)",
+    Explanation:
+      "All of these are current trends as of 2023. Having GDPR-compliant websites was a previous trend, while virtual realities provide the user with immersive and engaging experiences. SEO and ML continue to use alternative text and images on websites for people with visual impairments.",
+    Link: "https://www.hurix.com/6-web-accessibility-trends-for-your-business/",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:561]: we only accept links to freeCodeCamp resources like the new publication, or YouTube channel.
or the https://www.w3.org/

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -550,6 +550,16 @@ const accessibilityQuiz = [
       "WCAG version 1.0 was published on May 5, 1999 and was a total game-changer for web accessibility.",
     Link: "https://www.freecodecamp.org/news/web-accessibility-best-practices-and-checklist/",
   },
+  {
+    Question: "Which is not a current web accessibility trend?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:554]:
```suggestion
    Question: "Which of the following is NOT a current web accessibility trend?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts (hunk 1 above)
Comment (stmapman7) [reply src/data/accessibility-quiz.ts]: Got it - as these current trends are NOT listed on either resource, I will close this PR and reopen a new one.

**Pull request closed without merging**